views/pages/infoAcolhido.ejs

```diff
@@ -46,5 +46,8 @@
         </div>
     </div>
     <form action="<%= acolhido.id %>/prescricao" method="POST">
-        <input class = "botaoPrescricao" type="submit" value="Nova Prescrição">
+        <input class = "botao-prescricao" type="submit" value="Nova Prescrição">
     </form>
+    <form action="/acolhido/lista">
```

Comment:
Se a gente for usar `form` ao invés de `a`, é bom que a gente deixe explicito que o `METHOD` é um `GET`.